github_activity_tracker/sync/etag_cache.py

```
@@ -0,0 +1,86 @@
+# Optional TTL in seconds (default 7 days) to limit key growth
+DEFAULT_TTL = 7 * 24 * 3600
+
+
+def _redis_client():
+    """Return Redis client or None if unavailable."""
+    try:
+        import redis
+
+        url = getattr(settings, "GITHUB_ETAG_REDIS_URL", "redis://localhost:6379/1")
+        return redis.Redis.from_url(url, decode_responses=True)
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)